src/sidebar/screen/tree/ScreenTreeItem.ts

```diff
@@ -0,0 +1,13 @@
+import * as vscode from "vscode";
+import Screen from "../model/Screen";
+import TreeItem from "../../../common/vscode/tree/TreeItem";
+
+export default class ScreenTreeItem extends TreeItem {
+    public iconPath = this.screen.latestVersion.snapshot.url
+        ? vscode.Uri.parse(this.screen.latestVersion.snapshot.url, true)
+        : undefined;
```

Comment:
First one seems better to me, because when you read it you can understand it as "if url is filled, then use it", the way you suggest is shorter, but it is hard to understand I way meant while writing the code. 

Anyway, This is obsolete now, because I changed how it is used in another pr.

Reply:
In react or preact, the usage I have suggested is like a convention but since we are not writing a react code and you are the boss, however you want 😂